File: beecrowd/1_Beginner/1113_Ascending_and_descending.c
#include <stdio.h>
#include <math.h>

int main()
{


    while(1)
    {
        int x,y;
        scanf("%d %d",&x,&y);

        if(x==y)
        {
            break;
        }
        else
        {
            if(x>y)
            {
                printf("Decrescente\n");
            }
            else if(x<y)
            {
                printf("Crescente\n");
            }
        }

    }

    return 0;
}

File: beecrowd/1_Beginner/Python/1113_Ascending_and_Descending.py
while(True):
    row = list(map(int, input().split()))
    x = int(row[0])
    y = int(row[1])

    if x==y:
        break
    else:
        if x>y:
            print("Decrescente")
        elif x<y:
            print("Crescente")
